data/sea/50-psv.h

```diff
@@ -355,18 +355,19 @@ static psv_error_t psv_read_buffer (psv_state_t *s)
         const char  *value_end  = time_ptr - 1;
         const size_t value_size = value_end - value_ptr;
 
-        const bool new_entity = entity_cur_size != entity_size
-                             || memcmp (entity_cur, entity_ptr, entity_size) != 0;
+        const bool new_entity = memcmp (entity_cur, entity_ptr, entity_size) != 0;
```

Comment:
ah, okay. because entity_size *includes* the pipe, the size comparison is unnecessary?